packages/shared/providers.yaml

```diff
@@ -3700,6 +3700,31 @@ microsoft-power-bi:
         base_url: https://api.powerbi.com
     docs: https://docs.nango.dev/integrations/all/microsoft-power-bi
 
+mindbody:
+    display_name: Mindbody
+    categories:
+        - fitness
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://api.mindbodyonline.com/public/v6
+        headers:
+            API-Key: ${apiKey}
+            siteId: ${connectionConfig.siteId}
+        verification:
+            method: GET
+            endpoint: /site/locations
+    docs: https://docs.nango.dev/integrations/all/mindbody
+    credentials:
```

Comment:
Might be good to include `pattern` and `example`  for both `apiKey` and `siteId`, since this is tested.

Reply:
added examples